### ISSUE TITLE
Negative values represented as zero

### ISSUE THREAD

**Issue by Dev1lroot** [HeidiSQL/HeidiSQL#1379]
**Preconditions**

* HeidiSQL version: 11.2.0.6213
* Database type and version: MariaDB 10.4
* OS: Windows 10 x64 1909 (18363.1316)

**Describe the bug**

negative INTEGER values represented as 0

**To Reproduce**

1. Create table with integer field using another software or PMA and insert record with negative INT
2. Open this table and look at this record using HeidiSQL

**Screenshots or Crash reports**

**Comment by ansgarbecker** [comment]
Please post a screenshot, and even better, the CREATE TABLE code. I can't reproduce that here:

![grafik](https://user-images.githubusercontent.com/7986591/119706844-c4061880-be5a-11eb-9829-2dd9e0d6d20c.png)

**Comment by Dev1lroot** [comment]
did you set libmariadb.dll as library in Session manager, and btw I use dark theme (maybe matters)  The table rows i am looking on was originally created with zero values, and then they was modified by microservice, using:
> UPDATE `table` SET `field` = `field` - 1 WHERE condition
But the values I got are different in PhpMyAdmin and HeidiSQL.
It was like: I did that microservice and tested it, by the first I though that my microservice does not working, because there was no changes in HeidiSQL and then go to PMA and see that negative values are there, but HeidiSQL could not represent it even after refresh. I cannot provide the original table data, because it is a part of commercial product. but i can probably show the last fields (that was changed) and how they is in the HeidiSQL and PMA
